llama-index-packs/llama-index-packs-secgpt/examples/SecGPT.ipynb

```diff
@@ -0,0 +1,741 @@
+{
```

Comment:
<p>Is there a way to show the case when not using SecGPT or having these measures turned off? In other words, can we see the case when the attack is successful?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/13127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
That is a good idea! I have included another notebook, which defines a non-isolated LLM-based system to showcase the attack.